js/source/scripts.js

```diff
@@ -76,10 +76,10 @@
       // basic-main-menu.
       $(".js-nav-item-expandable li").unbind('click');
       // basic-main-menu.
-      $(".js-nav-item-expandable li a.parent").unbind('click').bind('click', function (e) {
+      $(".js-nav-item-expandable li a.parent").unbind('click').bind('click', function ( ) {
```

Comment:
by removing the `e` parameter you are disabling both `e.preventDefault()` and `e.stopPropagation()` which very well may be what you want but in that case you may as well remove them from this function as they don't do anything. Or, you could restore the `e` parameter and keep the name change from `event.stopPropagation()` to `e.stopPropagation()` as that might fix it as well.